subiquity/models/subiquity.py

```diff
@@ -23,6 +23,18 @@
 import uuid
 import yaml
 
+from cloudinit.config.schema import (
+        SchemaValidationError,
+        get_schema,
+        validate_cloudconfig_schema
+)
+
+try:
+    from cloudinit.config.schema import SchemaProblem
```

Comment:
SchemaProblem wasn't defined in original release on Bionic or Focal. So we'll hit import errors at runtime if subiquity is installing from the original released images. SchemaProblem is just a NamedTuple, and the original schema_errors were just a tuple, so we can create a function that does the same when handling `ImportError`